tests/e2e/framework/resource.go

```diff
@@ -155,7 +157,7 @@ func (ctx *Context) GetWatchNamespace() (string, error) {
 	return ctx.watchNamespace, nil
 }
 
-func (ctx *Context) createFromYAML(yamlFile []byte, skipIfExists bool, cleanupOptions *CleanupOptions) error {
```

Comment:
This method is only used to create objects from the global and namespaced manifest. This patch is explicitly handling cleanup for those files to ensure they're cleaned up in the proper order, which is why we don't need the cleanup options anymore.